components/keyboard_shortcuts/keyboard_shortcuts_sequence/keyboard_shortcuts_sequence.tsx

```diff
@@ -4,21 +4,23 @@ import {FormatXMLElementFn, PrimitiveType} from 'intl-messageformat';
 import React, {memo} from 'react';
 import {useIntl} from 'react-intl';
 
-import {isMessageDescriptor, KeyboardShortcutsSeq} from '../keyboard_shortcuts';
+import {ShortcutKetVariant, ShortcutKey} from '../../shortcut_key';
+
+import {isMac} from 'utils/utils';
+
+import {isMessageDescriptor, NormalizeKeyboardShortcutDescriptor} from './keyboard_shortcuts';
 
-import {isMac} from '../../../utils/utils';
 import './keyboard_shortcuts_sequence.scss';
-import {ShortcutKetVariant, ShortcutKey} from '../../shortcut_key';
 
 type Props = {
-    shortcut: KeyboardShortcutsSeq;
+    shortcut: NormalizeKeyboardShortcutDescriptor;
     values?: Record<string, PrimitiveType | FormatXMLElementFn<string, string>>;
     hideDescription?: boolean;
     hoistDescription?: boolean;
     isInsideTooltip?: boolean;
 };
 
-function normalize(shortcut: KeyboardShortcutsSeq) {
+function normalize(shortcut: NormalizeKeyboardShortcutDescriptor) {
```

Comment:
what I meant with me last review was to rename the function and not the type.

The type could very well stay as `KeyboardShortcutsSeq` (although I would probably rename that to `KeyboardShortcutsSequence`)

Reply:
Yeah you are right 🤦‍♂️.